spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/repository/config/EnableDatastoreRepositories.java

```diff
@@ -0,0 +1,137 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.datastore.repository.config;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.cloud.gcp.data.datastore.repository.support.DatastoreRepositoryFactoryBean;
+import org.springframework.context.annotation.ComponentScan.Filter;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.repository.config.DefaultRepositoryBaseClass;
+
+/**
+ * Annotation that enables the instantiation of Datastore repositories.
+ *
+ * @author Chengyuan Zhao
+ *
+ * @since 1.1
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Inherited
+@Documented
+@Import(DatastoreRepositoriesRegistrar.class)
+public @interface EnableDatastoreRepositories {
```

Comment:
Can we extend `org.springframework.data.repository.config.EnableRepositories`?

Reply:
@meltsufin I'm having trouble finding this file, but I did find this: 

https://github.com/spring-projects/spring-data-commons/blob/master/src/test/java/org/springframework/data/repository/config/EnableRepositories.java

Was this the one? This one seems to just be a test file used with the test "DummyRegistrar"

Reply:
Oops :)